rosdep/python.yaml

```
@@ -653,12 +653,21 @@ python-attrs:
     bionic: [python-attr]
     focal: [python-attr]
 python-attrs-pip:
+  arch:
+    pip:
+      packages: [attrs]
   debian:
     pip:
       packages: [attrs]
   fedora:
     pip:
       packages: [attrs]
+  osx:
+    pip:
+      packages: [attrs]
+  ubuntu:
+    pip:
+      packages: [attrs]
```

We can actually use an [OS wildcard](https://ros.org/reps/rep-0111.html#wildcard-os-name) here to make this rule cover all platforms rather than picking them individually.
```suggestion
  '*':
    pip:
      packages: [attrs]
```

---

Agreed. Thank you.